### DISCUSSION TITLE
Confusion over importance of Immer

### DISCUSSION THREAD

**Discussion by joeyfigaro** [udecode/zustand-x#52] [Q&A] [answered]
The documentation states that immer is required, I believe–but I don't see it listed as a dependency anywhere. Is it possible to use zustood _without_ immer?

**Comment by GoodbyeNJN** [comment, marked as answer]
You can see at [packages/zustood/package.json](https://github.com/udecode/zustood/blob/main/packages/zustood/package.json), immer has been listed as dependencies. So there is no way to separate immer from zustood
